src/open_anything_diffusion/datasets/flow_dataset_pyg.py

```diff
@@ -0,0 +1,59 @@
+from typing import List, Optional, Protocol, Union, cast
+
+import rpad.partnet_mobility_utils.dataset as pmd
+import torch
+import torch_geometric.data as tgd
+from flowbot3d.datasets.flow_dataset import Flowbot3DDataset
+
+
+class Flowbot3DTGData(Protocol):
+    id: str  # Object ID.
+
+    pos: torch.Tensor  # Points in the point cloud.
+    flow: torch.Tensor  # instantaneous positive 3D flow.
+    mask: torch.Tensor  # Mask of the part of interest.
+
+
+class Flowbot3DPyGDataset(tgd.Dataset):
```

Comment:
Since I updated upstream, this can probably be removed now right?

Reply:
Yes

Reply:
can you remove it from the change then? and incorporate the upstream one? to fetch upstream changes you may need to pip install the upstream again.

Reply:
Okay done